README.md

```diff
@@ -3,6 +3,7 @@
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
+
 - [📖 About the Project](#about-project)
```

Comment:
[Optional] Greate you use the Readme template. But some of the sections are not available in your Readme file. But section headings are listed in the "Table of content", I marked some of them with red color. Please consider removing them.

![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/55840999/e42e8a71-d999-41d2-951f-14273e28af25)